app/main/containers/Search/index.js

```diff
@@ -109,7 +110,9 @@ class Search extends Component {
     // NOTE: when page refreshed (location.reload) componentWillUnmount is not called
     window.addEventListener('beforeunload', this.cleanup)
     this.electronWindow.on('show', this.focusMainInput)
+    this.electronWindow.on('show', this.updateElectronWindow)
     this.electronWindow.on('show', trackShowWindow)
+    this.electronWindow.on('move', this.moveElectronWindow)
```

Comment:
All right, now the mainWindow is also initialized with correct position, and it uses the same fallback logic when screens have changed, etc.